src/cd-sensor.h

```diff
@@ -60,16 +62,20 @@ GQuark		 cd_sensor_error_quark		(void);
 /* accessors */
 const gchar	*cd_sensor_get_id		(CdSensor		*sensor);
 const gchar	*cd_sensor_get_object_path	(CdSensor		*sensor);
+#ifdef HAVE_UDEV
 const gchar	*cd_sensor_get_device_path	(CdSensor		*sensor);
+#endif
```

Comment:
I don't think this needs to be commented out in the header, just leave a stub in the .c file